### PR TITLE
Module 1: Portfolio mobile version

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
1. Added the index.html
2. Added the styles.css and linked it with index.html
3. Added a background color in place of the background image, because I couldn't find a bg image for my template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <link rel="stylesheet" href="style.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <a class="navbar-brand" href="/">Welcome</a>
+        <a class="navbar-hamburger" href="/">
+            <img src="./img/Union.png" alt="The hamburger menu" />
+        </a>
+    </nav>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please ensure that the container carrying the tool bar section is a `header tag`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <a class="navbar-brand" href="/">Welcome</a>
+        <a class="navbar-hamburger" href="/">
+            <img src="./img/Union.png" alt="The hamburger menu" />
+        </a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] only the hamburger should be inside the nav tag and not the logo.. 
```
<header>
    <p>logo<p/>
   <nav>hamburger<nav/>
<header/>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <a class="navbar-brand" href="/">Welcome</a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ optional] you can add your name in the place of `welcome` text since you don't have a logo yet

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <a class="navbar-hamburger" href="/">
+            <img src="./img/Union.png" alt="The hamburger menu" />
+        </a>
+    </nav>
+    <section class="header-section">
+        <div class="header-section-title">
+            <h2>Hey there. I’m Fadahunsi. <br/><span> I’m a software developer</span></h2>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please put the first title in an `h1 tag`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        <div class="header-section-text">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="header-section-icon-container">
+            <a href="/"><img src="./img/Linkedin.png" alt="The icons" /></a>
+            <a href="/"><img src="./img/angellist.png" alt="The icons" /></a>
+            <a href="/"><img src="./img/github.png" alt="The icons" /></a>
+            <a href="/"><img src="./img/medium.png" alt="The icons" /></a>
+            <a href="/"><img src="./img/twitter.png" alt="The icons" /></a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] Add the figma background to your headline section
your design; 👇🏾 
Screenshot (44)
figma design[ here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 👇🏾 
Screenshot (42)

Comment (tufoinnkuo10) [inline index.html]: - [ ] please put the icons in an `ul tag` and each one in a` li tag` ...
```
<ul>
    <li>icons<li/>
    <li>icons<li/>
<ul/>
```

File: style.css [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  color: #28352f;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  background-image: linear-gradient(to right, rgba(226, 235, 238, 0.62), rgba(195, 206, 200, 0.52));
```
Comment (tufoinnkuo10) [inline style.css]: - [ ]please use the figma background image and not color

File: style.css [hunk 7]
```diff
@@ -0,0 +1,62 @@
+* {
```
Comment (tufoinnkuo10) [inline style.css:1]: - [ ] please set your bottom margin well. it should look like the original version on figma

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Fadahunsiseyi 

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Good commit messages
✅ linter checks validated
✅ Good pull request summary
✅ Correct Readme template

Required Changes ♻️
Check the comments under the review

- [ ] please add a `.gitignore` file .  please also ensure that the file has your  `node_modules` inside so you don't accidentally push them to your remote repository

**Optional suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
```
Comment (Kakalanp) [inline index.html]: [OPTIONAL]

Excellent work implementing changes requested by the previous code reviewer! Just to add the cherry on top, you could remove the class and select this element by its `<header>` tag :cherries:

File: style.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./img/header-bg-mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```
Comment (Kakalanp) [inline style.css]: Kindly add some padding to this section to make it look more similar to the Figma template :wink:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @Fadahunsiseyi! hi

### Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Fadahunsiseyi ,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

- [ ] Kindly remove the line of text you are not using in the Readme file in this milestone.

Captureread

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**